server/plugin.go

```diff
@@ -77,3 +77,21 @@ func (p *Plugin) handleGetAttributes(w http.ResponseWriter, r *http.Request) {
 }
 
 // See https://developers.mattermost.com/extend/plugins/server/reference/
```

Comment:
I don't think this comment applies to this function

Reply:
Removed this comment